docs/toml-schema.md

```diff
@@ -306,4 +306,9 @@ dismiss-stale-review = false
 # This option is only relevant if bors is not used.
 # (optional - default `1`)
 required-approvals = 1
+# Which teams have access to push/merge to this branch.
```

Comment:
It's unclear from these instructions that these are *GitHub* teams and not the names of teams in the team repo. I think I'd prefer for it to be team repo teams and for us to do the mapping on behalf of the user, but in any case, we should document which is meant.

Reply:
Teams and GitHub teams are not 1:1, although we could probably put all GH teams of the team in there. But we already use GitHub teams for `[access.teams]`, so I think that we should keep it as that to be consistent. I modified the documentation.